src/data/community-events.json

```
@@ -232,5 +232,14 @@
     "description": "ETH Munich is a first time Hackathon organised by the local web3 community PretzelDAO",
     "startDate": "2023-08-11",
     "endDate": "2023-08-13"
+  },
+  {
+    "title": "Ethereum Community Conference",
```

```suggestion
    "title": "EthCC 6",
```
Going to change title to EthCC here as thats generally how the community refers to this event.